src/shared/clients/devenvClient.ts

```diff
@@ -11,7 +11,7 @@ import { getCodeCatalystDevEnvId } from '../vscode/env'
 import { ExtensionUserActivity } from '../extensionUtilities'
 
 const environmentAuthToken = '__MDE_ENV_API_AUTHORIZATION_TOKEN'
-const environmentEndpoint = 'http://127.0.0.1:1339'
+const environmentEndpoint = process.env['__MDE_ENVIRONMENT_API'] ?? 'http://127.0.0.1:1339'
```

Comment:
Ideally we would do some sanity checking (and log if necessary). MDE has had regressions where env vars were invalid or went missing.